modules/ip/datapath/gr_ip4_datapath.h

```diff
@@ -16,9 +16,7 @@
 
 #include <stdint.h>
 
-GR_MBUF_PRIV_DATA_TYPE(ip_output_mbuf_data, {
-	struct nexthop *nh;
-});
+GR_MBUF_PRIV_DATA_TYPE(ip_output_mbuf_data, { struct nexthop *nh; });
```

Comment:
Should this go in the previous commit?

Reply:
It seems so: I may not have run `make format` on the previous commit !